### ISSUE TITLE
Typescript compilation errors for keycloak plugin

### ISSUE THREAD

**Issue by awsjim** [janus-idp/backstage-plugins#1002]
### Describe the bug
I'm following the documentation to setup authn/authz with the janus-idp keycloak plugin as described at https://janus-idp.io/plugins/keycloak.
After installing the plugin and modifying the catalog plugin file, typescript compilation fails to find a declaration of types for the plugins

### Expected Behavior
`yarn tsc` should complete with no errors

### What are the steps to reproduce this bug?

1.  Install a fresh Backstage: `BACKSTAGE_APP_NAME=backstage npx -y -q @backstage/create-app@0.5.7 --path backstage`
2. Install the keycloak plugin: `cd backstage && yarn --cwd packages/backend add @janus-idp/backstage-plugin-keycloak-backend`
3. Modify the `packages/backend/src/plugins/catalog.ts` file as described at https://janus-idp.io/plugins/keycloak
```diff
import { CatalogBuilder } from '@backstage/plugin-catalog-backend';
import { ScaffolderEntitiesProcessor } from '@backstage/plugin-catalog-backend-module-scaffolder-entity-model';
import { Router } from 'express';
import { PluginEnvironment } from '../types';
+ import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';

export default async function createPlugin(
  env: PluginEnvironment,
): Promise<Router> {
  const builder = await CatalogBuilder.create(env);
  builder.addProcessor(new ScaffolderEntitiesProcessor());

+  builder.addEntityProvider(
+    KeycloakOrgEntityProvider.fromConfig(env.config, {
+      id: 'development',
+      logger: env.logger,
+      scheduler: env.scheduler,
+    }),
+  );

  const { processingEngine, router } = await builder.build();
  await processingEngine.start();
  return router;
}
```
4. Run the typescript compiler: `yarn tsc`

Typescript compilation results:
```
yarn run v1.22.21
$ tsc
packages/backend/src/plugins/catalog.ts:5:43 - error TS7016: Could not find a declaration file for module '@janus-idp/backstage-plugin-keycloak-backend'. '/Users/username/tmp/backstage/node_modules/@janus-idp/backstage-plugin-keycloak-backend/dist/index.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/janus-idp__backstage-plugin-keycloak-backend` if it exists or add a new declaration (.d.ts) file containing `declare module '@janus-idp/backstage-plugin-keycloak-backend';`

5 import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in packages/backend/src/plugins/catalog.ts:5

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Versions of software used and environment

- latest Backstage 1.20.3
- @janus-idp/backstage-plugin-keycloak-backend@1.

**Comment by mikkeschiren** [comment]
Confirmed here also.

**Comment by mikkeschiren** [comment]
Confirmed still missing in 1.9.9.

**Comment by youngyol** [comment]
It looks like the `index.d.ts` file is available from version 1.9.13. 
You can find more details and the latest version here: [npm package link](https://www.npmjs.com/package/@janus-idp/backstage-plugin-keycloak-backend/v/1.10.0?activeTab=code).
